### DISCUSSION TITLE
Any easy way to use lemonfox.ai's API url instead? it's only 1/3 of OpenAI's price and will be significant for me, thx

### DISCUSSION THREAD

**Discussion by ccchan234** [chidiwilliams/buzz#774] [General]
hi, lemonfox.ai's API is exactly the whisper api's,

adding this line 
openai.api_base = "https://api.lemonfox.ai/v1"  # Lemonfox.ai base URL
will make my previous python script that use OpenAI whisper API be able to use lemonfox.ai's (i dont have GPU thus wanna process using the companys' servers)

it's only 1/3 the price of OpenAI.

thank you.

ref:
https://www.lemonfox.ai/apis

**Comment by ccchan234** [comment]
ps: looks like not 100% same syntax /w OpenAI's API.
need some manipulatoin.

but it's just 1 - 2 lines.

if author would point out how,
other could try to do so, thanks

**Comment by raivisdejus** [comment]
Latest development versions have option to specify custom OpenAI API base url. Log into GitHub and download the latest development version from some successful Action https://github.com/chidiwilliams/buzz/actions
